boudicca.base/query-lib/src/main/kotlin/base/boudicca/query/BoudiccaQueryBuilder.kt

```diff
@@ -45,17 +43,14 @@ object BoudiccaQueryBuilder {
         return escapeText(dateFieldName) + " before " + escapeText(DateTimeFormatter.ISO_LOCAL_DATE.format(localDate))
     }
 
+    @Suppress("detekt:ExceptionRaisedInUnexpectedLocation")
```

Comment:
why supress when fixed?

Reply:
it is not fixed. the method is called "equals" because the query is called like that. but normally "equals" methods have a different contract, you know like comparing objects and are not supposed to throw exceptions. this is a different kind of equals method so i suppress the rule